modules/perception/onboard/component/BUILD

```diff
@@ -3,6 +3,55 @@ load("//tools:cpplint.bzl", "cpplint")
 
 package(default_visibility = ["//visibility:public"])
 
+cc_binary(
+    name = "libperception_component_camera.so",
+    linkshared = True,
+    linkstatic = False,
+    deps = [":perception_component_inner_camera"],
+)
+
+cc_library(
+    name = "perception_component_inner_camera",
+    srcs = [
+        "trafficlights_perception_component.cc",
+    ],
+    hdrs = [
+        "trafficlights_perception_component.h",
+    ],
+    copts = [
+        '-DMODULE_NAME=\\"perception\\"',
+    ],
```

Comment:
Define a `PERCEPTION_COPTS = [  '-DMODULE_NAME=\\"perception\\"' ]` line after the `package(...)` line, 

and use it in this BUILD file where needed:

```
copts = PERCEPTION_COPTS,
```